packages/next/src/cli/next-dev.ts

```diff
@@ -373,7 +377,29 @@ const nextDev: CliCommand = async (argv) => {
     const runDevServer = async (reboot: boolean) => {
       try {
         const workerInit = await createRouterWorker()
-        await workerInit.worker.startServer(devServerOptions)
+        if (!!args['--experimental-https']) {
+          let certificate: { key: string; cert: string } | undefined
+
+          if (
+            args['--experimental-https-key'] &&
+            args['--experimental-https-cert']
+          ) {
+            certificate = {
+              key: path.resolve(args['--experimental-https-key']),
```

Comment:
Consider allowing to drive these tw flag's value by an environment variable? 
Then you could have one global environment variable in a monorepo for all Next.js applications

Reply:
We will consider this once the feature isn’t experimental 👍

Reply:
Thank you, that makes sense. 
As an example in my current four Next.js in our monorepo I am doing this in the custom server:
```typescript
createServer({
    certificatePath: getCertificatePath('crt.pem'),
    certificateKeyPath: getCertificatePath('key.pem'),
    caCertificatePath: getCertificatePath('ca.pem'),
})

export function getCertificatePath(file: string) {
  const certificatesBasePath = process.env.CERTIFICATES_PATH ?? ''
  const certificatesSourcePath =
    certificatesBasePath.length > 0 ? certificatesBasePath : path.join(process.cwd(), 'tls')
  return path.join(certificatesSourcePath, file)
}
```